UP_SAT:   100
DOWN_SAT: -100

K_P:
  x: 4.9
  y: 5
  z: 5
  roll: 0
  pitch: 2.5
  yaw: 12.2
  ni_1.x: 200

K_I:
  x: 0
  y: 0.1
  z: 0.1
  roll: 0
  pitch: 1.6
  yaw: 0.04
  ni_1.x: 0.06



